pex/util.py

```diff
@@ -213,3 +215,32 @@ def named_temporary_file(*args, **kwargs):
       yield fp
   finally:
     os.remove(fp.name)
+
+
+def iter_pth_paths(filename):
+  """Given a .pth file, extract and yield all inner paths. Largely lifted from site.py."""
+  dirname = os.path.dirname(filename)
+  known_paths = set()
+
+  try:
+    f = open(filename, 'rU')  # noqa
+  except IOError:
+    return
```

Comment:
I can see why `site.py` does this, but is this still what we want to do in our case?